apps/www/data/career.json

```diff
@@ -55,7 +55,6 @@
     "mikqi",
     "florian-lefebvre",
     "m4salah",
-    "carlosstenzel",
```

Comment:
Who is this BTW?

Reply:
<img height="100" src="https://github.com/carlosstenzel.png">

https://github.com/carlosstenzel

Reply:
Is the reason for removing because we can only have so many contributors shown in the page?

Reply:
Just asking, because I don't know how the website works!

Reply:
The figma design had a max of 18, so I just went to off of that. but I can add one more in the top right corner without removing anyone.